### PR TITLE
Part 2: New Documentation

### PR DESCRIPTION
**Pull request by kitten**
Related to #548

### CODE REVIEW

File: docs/basics/queries.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+const Todos = () => {
+  const [result, reexecuteQuery] = useQuery({
+    query: TodosQuery,
+  });
+
+  const { data, fetching, error } = result;
+
+  if (fetching) {
+    return <p>Loading...</p>;
+  } else if (error) {
```
Comment (andyrichardson) [inline docs/basics/queries.md]: Need to get rid of this unnecessary else - there's quite a few in the codebase also

```suggestion
  }
  
  if (error) {
```

Comment (kitten) [reply docs/basics/queries.md]: I’m curious, do you think it’s an issue with readability? I assumed since it’s equivalent both is fine 🤗

Comment (andyrichardson) [reply docs/basics/queries.md]: Yes I think adding unnecessary conditional logic an issue with readability

**Comment by kitten** [comment]
The staging deployment seems to be working now. Here’s the link to this PR’s deployed docs 🌟 https://formidable-com-urql-staging-556.surge.sh/docs

**Pull request merged**